bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/icabanken/ICABanken.java

```diff
@@ -130,11 +124,6 @@ public void update() throws BankException, LoginException,
 
     }
 
-    @Override
```

Comment:
This is just an empty method and the super method in Bank is not abstract so this code is redundant.